command/server/server_command.go

```diff
@@ -211,6 +211,14 @@ func (c *ServerCommand) DefineFlags() {
 		},
 		FlagOptional: true,
 	}
+
+	c.FlagMap["block-time"] = helper.FlagDescriptor{
+		Description: "Sets block time in miliseconds. Default: 2000",
```

Comment:
Is it possible to specify default block time in this description from default config, not hardcoded?

Reply:
Default value added from default config 631515404cf370d1fc015938ea35da52ff9dbaaf